Zend/zend_vm_gen.php

```diff
@@ -779,145 +779,83 @@ function gen_code($f, $spec, $kind, $export, $code, $op1, $op2, $name, $extra_sp
-			$op1_get_zval_ptr_ptr_undef[$op1],
-			$op2_get_zval_ptr_ptr_undef[$op2],
-			$op1_get_obj_zval_ptr[$op1],
-			$op2_get_obj_zval_ptr[$op2],
-			$op1_get_obj_zval_ptr_undef[$op1],
-			$op2_get_obj_zval_ptr_undef[$op2],
-			$op1_get_obj_zval_ptr_deref[$op1],
-			$op2_get_obj_zval_ptr_deref[$op2],
-			$op1_get_obj_zval_ptr_ptr[$op1],
-			$op2_get_obj_zval_ptr_ptr[$op2],
-			$op1_get_obj_zval_ptr_ptr_undef[$op1],
-			$op2_get_obj_zval_ptr_ptr_undef[$op2],
-			$op1_free_op[$op1],
-			$op2_free_op[$op2],
-			$op1_free_op_if_var[$op1],
-			$op2_free_op_if_var[$op2],
-			$op1_free_op_var_ptr[$op1],
-			$op2_free_op_var_ptr[$op2],
-			$op1_free_unfetched[$op1],
-			$op2_free_unfetched[$op2],
-			($op1!="ANY"||$op2!="ANY")?"#\\1if 1\n":"#\\1if 0\n",
-			($op1!="ANY"||$op2!="ANY")?"#\\1if 0\n":"#\\1if 1\n",
-			($op1!="ANY"||$op2!="ANY")?"0":"1",
-			($op1!="ANY"||$op2!="ANY")?"1":"0",
-			"\\1".(($spec && $kind != ZEND_VM_KIND_CALL)?("_SPEC".$prefix[$op1].$prefix[$op2].extra_spec_name($extra_spec)):""),
-			"goto \\1".(($spec && $kind != ZEND_VM_KIND_CALL)?("_SPEC".$prefix[$op1].$prefix[$op2].extra_spec_name($extra_spec)):""),
-			"#\\1if 1",
-			"#\\1if 0",
-			$export?"#\\1if 1\n":"#\\1if 0\n",
-			$export?"#\\1if 0\n":"#\\1if 1\n",
-			$op_data_type[isset($extra_spec['OP_DATA']) ? $extra_spec['OP_DATA'] : "ANY"],
-			$op_data_get_zval_ptr[isset($extra_spec['OP_DATA']) ? $extra_spec['OP_DATA'] : "ANY"],
-			$op_data_get_zval_ptr_deref[isset($extra_spec['OP_DATA']) ? $extra_spec['OP_DATA'] : "ANY"],
-			$op_data_get_zval_ptr_ptr[isset($extra_spec['OP_DATA']) ? $extra_spec['OP_DATA'] : "ANY"],
-			$op_data_free_op[isset($extra_spec['OP_DATA']) ? $extra_spec['OP_DATA'] : "ANY"],
-			$op_data_free_op_var_ptr[isset($extra_spec['OP_DATA']) ? $extra_spec['OP_DATA'] : "ANY"],
-			$op_data_free_unfetched[isset($extra_spec['OP_DATA']) ? $extra_spec['OP_DATA'] : "ANY"],
-			isset($extra_spec['RETVAL']) ? $extra_spec['RETVAL'] : "RETURN_VALUE_USED(opline)",
-			isset($extra_spec['QUICK_ARG']) ? $extra_spec['QUICK_ARG'] : "arg_num <= MAX_ARG_FLAG_NUM",
-			isset($extra_spec['SMART_BRANCH']) ?
-				($extra_spec['SMART_BRANCH'] == 1 ?
-						"ZEND_VM_SMART_BRANCH_JMPZ(\\1, \\2)"
-					:	($extra_spec['SMART_BRANCH'] == 2 ?
-							"ZEND_VM_SMART_BRANCH_JMPNZ(\\1, \\2)" : ""))
-				: 	"ZEND_VM_SMART_BRANCH(\\1, \\2)",
-			isset($extra_spec['DIM_OBJ']) ?
-				($extra_spec['DIM_OBJ'] == 0 ? "1" : "0")
-				: "\\0",
-			isset($extra_spec['DIM_OBJ']) ?
-				($extra_spec['DIM_OBJ'] == 1 ? "1" : "0")
-				: "\\0",
-			isset($extra_spec['DIM_OBJ']) ?
-				($extra_spec['DIM_OBJ'] == 2 ? "1" : "0")
-				: "\\0",
-			isset($extra_spec['DIM_OBJ']) ?
-				($extra_spec['DIM_OBJ'] == 3 ? "1" : "0")
-				: "\\0",
-			isset($extra_spec['ISSET']) ?
-				($extra_spec['ISSET'] == 0 ? "0" : "1")
-				: "\\0",
-			isset($extra_spec['ISSET']) ?
-				($extra_spec['ISSET'] == 0 ? "\\0" : "opline->extended_value")
-				: "\\0",
-		),
-		$code);
+	$specialized_replacements = array(
```

Comment:
Variable never used?

Reply:
@nikic fixed, missed it when cleaning up.